docs/scripts_md/file_insertion.md

```diff
@@ -0,0 +1,83 @@
+# NAME
```

Comment:
No sure if this file is in sync with the latest modifications you made but at the very least it should be named `imaging_non_minc_insertion.md`.

Reply:
@nicolasbrossard Oups... Recreated the markdown file. Also renamed it in the mass perldoc script.